### DISCUSSION TITLE
오류 사항 올립니다

### DISCUSSION THREAD

**Discussion by BBungya** [HeoJiye/Github-Study#12] [질문 게시판] [answered]
내용대로 따라했는데 커밋에서 자꾸 오류가 납니다
![image](https://user-images.githubusercontent.com/122617471/214245914-49fef32f-a898-49fc-a3f8-b040def6944b.png)

**Comment by HeoJiye** [comment, marked as answer]
깃을 사용하려면 깃에 이름과 이메일을 먼저 등록해야 합니다!

[Git & GitHub 기초 지식.zip>Git 설정](https://github.com/HeoJiye/Github-Study/blob/main/note/github-basic-zip.md#git-%EC%84%A4%EC%A0%95)
이 곳 참고 부탁드립니다. 등록 후, 또 오류가 나면 댓글 남겨주세요
